python/paddle/fluid/layers/nn.py

```diff
@@ -272,7 +272,9 @@ def dynamic_lstm(input,
                  cell_activation='tanh',
                  candidate_activation='tanh',
                  dtype='float32',
-                 name=None):
+                 name=None,
+                 h_0=None,
+                 c_0=None):
```

Comment:
Put the name at the end?

Remove the TODO in line 264.

Reply:
Done.